vllm_ascend/platform.py

```diff
@@ -86,6 +86,14 @@ def get_device_capability(cls, device_id: int = 0):
     def get_device_name(cls, device_id: int = 0) -> str:
         return torch.npu.get_device_name(device_id)
 
+    @classmethod
+    def get_device_count(cls) -> int:
```

Comment:
1. there is no interface for these two action. https://github.com/vllm-project/vllm/blob/main/vllm/platforms/interface.py it's not suggested to add a new func here.
2. after https://github.com/vllm-project/vllm/commit/7272bfae77310c417e0b747a0590d9dd782c9f6b this commit. now vllm can use current_platform.device_count() directlly.